pkg/tokens/tokens.go

```diff
@@ -0,0 +1,24 @@
+package tokens
+
+import (
+	"time"
+
+	"github.com/Solidform-labs/newsletter/configs"
+	"github.com/golang-jwt/jwt/v5"
+)
+
+func CreateToken(email string) (string, error) {
+	token := jwt.NewWithClaims(jwt.SigningMethodHS256, jwt.MapClaims{
+		"email": email,
+		"exp":   time.Now().Add(time.Second * 10).Unix(),
```

Comment:
Change expiration time to either be something generally sensible (10 minutes, for example) or configurable through an environment variable.